pkg/cri/server/container_remove.go

```diff
@@ -106,6 +112,14 @@ func (c *criService) RemoveContainer(ctx context.Context, r *runtime.RemoveConta
 
 	c.containerNameIndex.ReleaseByKey(id)
 
+	event := runtime.ContainerEventResponse{
+		ContainerId:        id,
+		ContainerEventType: runtime.ContainerEventType_CONTAINER_DELETED_EVENT,
+		CreatedAt:          time.Now().UnixNano(),
+		PodSandboxMetadata: sandbox.Config.Metadata,
```

Comment:
per discussion let's figure out what status data about the pod is actually needed to keep state in sync for the pod vs having kubelet call down to get pod status with each container event ..

Reply:
for ref, `PodSandboxMetata` is used by kubelet here: https://github.com/kubernetes/kubernetes/pull/111384/files#diff-465aae123f6877816ff0428968fc74f503fe15b27b434480bf57b79e62c7fa2fR286

```
status, err := e.runtime.GetPodStatus(podID, podName, podNamespace)
```

Reply:
right.. hoping that was just a temporary test/change and we can maybe go back to the KEP? or refresh why more is needed than the podsandbox id per container event..
```
// ID of the sandbox container
     string sandbox_id = 4;
```

if more is needed sure let's figure out how best to do it in a single event